src/Compilers/Core/Portable/CodeAnalysisEventSource.Common.cs

```diff
@@ -13,6 +13,7 @@ public static class Keywords
         {
             public const EventKeywords Performance = (EventKeywords)1;
             public const EventKeywords Correctness = (EventKeywords)2;
+            public const EventKeywords AnalyzerLoading = (EventKeywords)4;
```

Comment:
Why did we skip three here? If this is meant to be a bitfield consider using the binary literals to make it clearer.

Reply:
Yeah, ETW requires them to be powers of two. I'll update it.